### PR TITLE
several improvements

### PR DESCRIPTION
**Pull request by gdams**
closes: https://github.com/AdoptOpenJDK/blog/issues/4
closes: https://github.com/AdoptOpenJDK/blog/issues/3
closes: https://github.com/AdoptOpenJDK/blog/issues/2

### CODE REVIEW

File: content/authors.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
   "georgeadams": {
     "name": "George Adams",
     "summary": "Java Program Manager @microsoft, Chair of TSC @AdoptOpenJDK, Core Collaborator @nodejs",
-    "twitter": "gdams",
-    "github": "gdams_"
+    "twitter": "gdams_",
+    "github": "gdams"
```
Comment (aahlenst) [inline content/authors.json:24]: 🤦

File: gatsby-node.js [hunk 2]
```diff
@@ -4,6 +4,22 @@ const { createFilePath } = require(`gatsby-source-filesystem`)
 exports.createPages = async ({ graphql, actions }) => {
   const { createPage } = actions
 
+  const authorJson = require(`./content/authors.json`)
+  const authorPage = path.resolve(`./src/templates/author-page.js`)
+
+  for (let author of Object.keys(authorJson)) {
+    console.log(author)
```
Comment (aahlenst) [inline gatsby-node.js:11]: Remove

File: src/components/byline.js [hunk 3]
```diff
@@ -1,10 +1,24 @@
...
+    <div
+      style={{
+        display: `flex`,
+        alignItems: `center`,
+        marginTop: rhythm(1 / 2),
+        marginBottom: rhythm(1 / 2),
+        textDecoration: `none`
+      }}
+    >
+    {date} – posted by &nbsp;<a href={href}>{author}</a> <ProfilePicInline identifier={identifier} name={author.name} />
```
Comment (aahlenst) [inline src/components/byline.js:20]: I guess this should be `<Link>`, not `<a>`.

File: src/components/layout.js [hunk 4]
```diff
@@ -27,7 +28,14 @@ const Layout = ({ location, title, children }) => {
   return (
     <div>
       <header id="site-header-main">
+        <link
+          rel="stylesheet"
+          href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+        />
```
Comment (aahlenst) [inline src/components/layout.js:34]: We already have Font Awesome in use via https://github.com/FortAwesome/react-fontawesome. In `layout.js`, you can declare what you need and then reference them inline.

File: src/components/socialbar.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+const SocialBar = () => {
+  return (
+    <>
+      <div className="social-icon align-center inline-block">
+        <a
+          href="https://twitter.com/adoptopenjdk"
+          target="_blank"
+          className="light-link no-underline"
+        >
```
Comment (aahlenst) [inline src/components/socialbar.js:12]: There are multiple warnings on the page concerning `<a>` tags:

```
 warning  Using target="_blank" without rel="noopener noreferrer" is a security risk: see
https://mathiasbynens.github.io/rel-noopener
```

File: src/components/socialbar.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+          <span className="fa-stack fa-lg" style={{ fontSize: "1.10em" }}>
+            <i className="fa fa-twitter fa-stack-1x fa-social-icon" />
```
Comment (aahlenst) [inline src/components/socialbar.js:14]: Please replace with `<FontAwesomeIcon>`.

File: src/components/socialbar.js [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+        </a>
+      </div>
+      <div className="social-icon align-center inline-block">
+        <a
+          href="https://www.facebook.com/adoptopenjdk"
+          target="_blank"
+          className="light-link no-underline"
+        >
+          <span className="fa-stack fa-lg" style={{ fontSize: "1.10em" }}>
+            <i className="fa fa-facebook fa-stack-1x fa-social-icon" />
```
Comment (aahlenst) [inline src/components/socialbar.js:25]: Please replace with `<FontAwesomeIcon>`.

File: src/components/socialbar.js [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+        </a>
+      </div>
+      <div className="social-icon align-center inline-block">
+        <a
+          href="https://www.youtube.com/c/AdoptOpenJDK"
+          target="_blank"
+          className="light-link no-underline"
+        >
+          <span className="fa-stack fa-lg" style={{ fontSize: "1.10em" }}>
+            <i className="fa fa-youtube-play fa-stack-1x fa-social-icon" />
```
Comment (aahlenst) [inline src/components/socialbar.js:36]: Please replace with `<FontAwesomeIcon>`.

File: src/components/socialbar.js [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+        </a>
+      </div>
+      <div className="social-icon align-center inline-block">
+        <a
+          href="https://www.twitch.tv/adoptopenjdk"
+          target="_blank"
+          className="light-link no-underline"
+        >
+          <span className="fa-stack fa-lg" style={{ fontSize: "1.10em" }}>
+            <i className="fa fa-twitch fa-stack-1x fa-social-icon" />
```
Comment (aahlenst) [inline src/components/socialbar.js:47]: Please replace with `<FontAwesomeIcon>`.

**Comment by gdams** [comment]
@aahlenst updated PTAL

**Comment by aahlenst** [comment]
Closed via 609aba009179f379a83fbe73c5dfb18dacaccbc1

**Pull request closed without merging**